### PR TITLE
Set schema sql removed

### PR DESCRIPTION
**Pull request by dheepak-aot**
The following item has been mentioned in the PR 1036 (https://github.com/bcgov/SIMS/pull/1036) but the code snippet was missed to be pushed.

image

### CODE REVIEW

**Review by henryoforeh-aot — APPROVED** [review]

File: sources/packages/api/src/database/database.service.ts [hunk 1]
```diff
@@ -8,13 +8,5 @@ export class DatabaseService {
...
-  constructor(public connection: Connection) {
-    connection
-      .query(`SET SCHEMA '${process.env.DB_SCHEMA || "sims"}';`)
-      .then(() => {
-        this.logger.log(
-          `*** Successfully set schema ${process.env.DB_SCHEMA || "sims"}`,
-        );
-      });
-  }
+  constructor(public connection: Connection) {}
```
Comment (andrewsignori-aot) [inline sources/packages/api/src/database/database.service.ts:11]: The logger can be removed also.

**Review by ann-aot — APPROVED** [review]

File: sources/packages/api/src/database/database.service.ts (hunk 1 above)
Comment (dheepak-aot) [reply sources/packages/api/src/database/database.service.ts:6]: Done.

**Review by guru-aot — APPROVED** [review]
LGTM

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Pull request merged**